File: SiteDemo/Test.WEB/src/app/_models/question.ts
export class Question {
  id: string;
  questionText: string;
  language: number;
  option1: string;
  option2: string;
  option3: string;
  option4: string;
  questionPhoto: string;
  answerNum: number;

  constructor(id?: string, questionText?: string, language?: number, option1?: string, option2?: string, option3?: string, option4?: string, questionPhoto?: string, answerNum?: number) {
    this.id = id;
    this.questionText = questionText;
    this.language = language;
    this.option1 = option1;
    this.option2 = option2;
    this.option3 = option3;
    this.option4 = option4;
    this.questionPhoto = questionPhoto;
    this.answerNum = answerNum;
  }
}

File: SiteDemo/Test.MODELS/Entities/Question.cs
using Test.MODELS.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Test.MODELS.DTO;
using Test.MODELS.Enums;

namespace Test.MODELS.Entities
{
    public class Question : IEntity
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }

        public string QuestionText { get; set; }

        public string QuestionPhoto { get; set; }

        public int Language { get; set; }

        public string Option1 { get; set; }

        public string Option2 { get; set; }

        public string Option3 { get; set; }

        public string Option4 { get; set; }

        public int AnswerNum { get; set; }
    }
}
